.github/workflows/build-and-push.yml

```diff
@@ -0,0 +1,25 @@
+name: Build to release branches
+
+on:
+  workflow_dispatch:
+  push:
+    tags: ['*']
+    branches:
+      - dev-beta
+      - dev-main
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: false
+
+jobs:
+  build-assets:
+    uses: inpsyde/reusable-workflows/.github/workflows/build-and-push-assets.yml@main
+    with:
+      PHP_VERSION: '8.0'
```

Comment:
Most probably irrelevant, but does it make sense to match the PHP version used at the target environemtn (IIRC that would be 8.2).

Reply:
Sorry, this was my fault. I remembered it the other way around in https://github.com/RRZE-Webteam/FAU-Studium/pull/125#discussion_r1584267363. 🤦🏽
But of course, in GitHub Actions workflows, the `PHP_VERSION` input must match the target server version ([documentation](https://inpsyde.atlassian.net/wiki/spaces/DEV3/pages/4662427763/Update+PHP+versions+in+packages#:~:text=In%20Github%20Actions%20workflows%2C%20the%20PHP_VERSION%20input%20must%20match%20the%20target%20server%20version)).